docs/pages/branding/about.tsx

```diff
@@ -59,9 +59,8 @@ function Image(props: ImageProps) {
 function BrandingHero() {
   return (
     <Container>
-      <Typography variant="h1" align="center" sx={{ mt: 9 }}>
-        We&apos;re on a mission to make building UIs with <UnderlinedText>React</UnderlinedText>{' '}
-        fun.
+      <Typography variant="h1" align="center" sx={{ mt: 9, maxWidth: 870, mx: 'auto' }}>
```

Comment:
How does this look with a changed browser font size i.e. would `rem` make more sense instead of `px`?

Reply:
Oh, do you mean that in the configuration of the base theme, we forgot to use rem instead of px? Yeah, looks about right: 

https://github.com/mui-org/material-ui/blob/d2ef73aee239655421a9bcd2ad50f5b7844541ca/docs/src/modules/branding/BrandingRoot.tsx#L117

In this very case, I think that no matter the base font size, we need a max-width in pixels, and not ch, as the contractions is on the spacing aesthetic. But I could be wrong.

Reply:
My question is how it looks if you double the browser font size (settings > appearance > font size > very large).

Reply:
> My question is how it looks if you double the browser font size (settings > appearance > font size > very large).

In the current configuration, it's invariant to appearance font size settings

Reply:
"current" referring to the prior state or the one in this PR? Both cases are bad though it's not obvious why the prior state would not scale with the browser font size since Typography uses `rem`.

Reply:
"current" refers to the prior state and this PR. Font sizes are in pixels, here.

Reply:
Then this is another issue that must be resolved. The docs work fine so I don't understand why this was changed for the branding.

Reply:
Agree, please see the latest commits :)

Reply:
Just looked at the site and font scaling is not ok. There are so many text containers that have a pixel based max-width.

Reply:
I have used the ch unit where it seemed to improve the rendering. However, I still had to account for the font-family as ch doesn't solve this problem: https://meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit/.

Reply:
> I still had to account for the font-family as ch doesn't solve this invariance problem: meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit.

I know what the `ch` unit is. Could you explain in your own words what the problem is? Why is invariance a problem here?

Reply:
My previous comment was meant to signal others that I tested that the layout shift was still fixed after the change of unit.  

> I know what the ch unit is.

?

> Could you explain in your own words what the problem is? Why is invariance a problem here?

The problem is the one in the pull request's description. During the rendering cycle of the page, two different fonts are used. For a given ch value, the text can wrap at a different word for a different font family. The value simply needs to be picked to be compatible with out fallback fonts.

Reply:
So we have a flash of unstyled text (FOUT). I think the most robust solution is either using system fonts or choosing a better fallback font that is closer to the original one. Either way, we can't "fix" this. We can only reduce it.

> > I know what the ch unit is.
> 
> ?

You were posting an article that explains what the `ch` unit is. Unless you think I don't know what the `ch` unit is I fail to see the relevance.

Reply:
Yeah, maybe we could change the font family approach. Since we already have a custom font, the system font is only relevant during the small time Inter isn't loaded yet. In which case, we can pick a fallback font that is widely available and close to the targeted one.

I'm looking at it.

Reply:
I can't find any easy solution. https://github.com/rsms/inter/issues/160 didn't help much. Ideally would use this tool https://meowni.ca/font-style-matcher/ and apply the correct CSS properties based on the currently loaded font. But the how-to implement it seems not worth the effort. 

I wouldn't be against: 

```diff
diff --git a/docs/pages/_app.js b/docs/pages/_app.js
index f86c107986..6084a1505f 100644
--- a/docs/pages/_app.js
+++ b/docs/pages/_app.js
@@ -303,7 +303,7 @@ function AppWrapper(props) {

   let fonts = [
     'https://fonts.googleapis.com/css?family=Roboto:300,400,400italic,500,700&display=swap',
-    'https://fonts.googleapis.com/css?family=Inter:400,600,700&display=swap',
+    'https://fonts.googleapis.com/css?family=Inter:400,600,700&display=block',
   ];
   if (router.pathname.match(/onepirate/)) {
     fonts = [
```

Reply:
Actually, we can ignore the font issue and force a specific word to break the line. Simpler.

Reply:
done in 9e5e8a8